android/app/src/main/res/values/strings.xml

```diff
@@ -344,4 +344,11 @@
 
     <string name="mobile_data_pref_warning">Wifi not found. Do you want to download using Mobile Data?</string>
 
+
+    <string name="summit_2018">Summit 2018</string>
+    <string name="starts_on">Starts On</string>
+    <string name="ends_on">Ends On</string>
+    <string name="organizer_details">Organizer Details</string>
```

Comment:
I think this was Organiser Description earlier

Reply:
Okay :+1: